src/App.tsx

```diff
@@ -1,61 +1,146 @@
+import { useState } from 'react';
+
 import './App.scss';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const modifiedTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(modifiedTodos);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
+  const [userIdError, setUserIdError] = useState(false);
+  const [titleErrorValidation, setTitleErrorValidation] = useState(false);
```

Comment:
Name the boolean values according to these rules to make your code cleaner and more understandable for other developers:
![image](https://user-images.githubusercontent.com/96409563/212643071-ab9a75cd-a108-439a-9ffe-0ee0b1e46a48.png)

Reply:
Thank you,
I renamed boolean values in my code and will use this rule in next tasks